ensime_shared/config.py

```diff
@@ -7,7 +7,7 @@
 
 from ensime_shared.util import Util
 
-BOOTSTRAPS_ROOT = os.path.join(os.environ['HOME'], '.config/ensime-vim/')
+BOOTSTRAPS_ROOT = os.path.join(os.environ['HOME'], '.config', 'ensime-vim')
```

Comment:
Unrelated, but the config folder location could be improved by using XDG for Linux and AppData for Windows.